### PR TITLE
[docs] Add status header for dynatrace exporter

### PR DESCRIPTION
**Pull request by Frapschen**
**Description:** <Describe what has changed.>
Add status header for dynatrace exporter

### CODE REVIEW

**Review by dashpole — APPROVED** [review]

File: exporter/dynatraceexporter/README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Dynatrace Exporter
 
+| Status                   |           |
+| ------------------------ |-----------|
+| Stability                | [beta]    |
+| Supported pipeline types | metrics   |
+| Distributions            | [contrib] |
```
Comment (dyladan) [inline exporter/dynatraceexporter/README.md:7]: We are also in the aws distro. Should that be listed here?

**Review by TylerHelmuth — APPROVED** [review]

File: exporter/dynatraceexporter/README.md (hunk 1 above)
Comment (TylerHelmuth) [reply exporter/dynatraceexporter/README.md:7]: I think that makes sense.

Comment (TylerHelmuth) [reply exporter/dynatraceexporter/README.md:7]: @codeboten how do you feel about linking to non-opentelemetry distributions?

Comment (TylerHelmuth) [reply exporter/dynatraceexporter/README.md:7]: @Frapschen please add the AWS distribution

**Review by dyladan — APPROVED** [review]

**Review by codeboten — APPROVED** [review]

**Pull request merged**